docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.rst

```diff
@@ -140,6 +140,24 @@ This guide will explain how to host and share Data Docs on Google Cloud Storage.
 
   Issue the following CLI command: ``gcloud app browse``. If successful, the gcloud CLI will provide the URL to your app and launch it in a new browser window. The page displayed should be the index page of your Data Docs site.
 
+
+**Additional notes**
+
+- If you wish to host a Data Docs site through a private DNS, you can configure a ``base_public_path`` for the Data Docs Store.  The following example will configure a GCS site with the ``base_public_path`` set to ``www.mydns.com``.
```

Comment:
This is the current version of the documentation for hosting Data Docs through a private DNS server using the `base_public_path` parameter.   Please let me know what you think, and what can be taken away / added. 

Since this affects the TupleGCSStoreBackend and TupleS3StoreBackend, a similar note would also need to go to the Validation Store and Expectation Store documentation

Reply:
@Aylr please let me know what you think of this section